src/main/java/de/retest/recheck/review/ignore/io/Loaders.java

```diff
@@ -72,9 +74,7 @@ private Loaders() {}
 	}
 
 	public static Stream<?> load( final Stream<String> lines ) {
-		return lines.map( String::trim ) //
-				.filter( StringUtils::isNotEmpty ) //
-				.map( Loaders::load );
+		return lines.map( Loaders::load );
```

Comment:
We need to take care that users know that they should not indent their rules, else there will be an exception. Or we could add a loader that, in the end, tries to trim and delegates to another loader.